Sources/Logging/Strings/StoreKitStrings.swift

```diff
@@ -200,6 +208,22 @@ extension StoreKitStrings: LogMessage {
 
         case let .error_displaying_store_message(error):
             return "Error displaying StoreKit message: '\(error)'"
+
+        case let .error_displaying_offer_code_redemption_sheet(error):
+            return "Error displaying Offer Code redemption sheet: '\(error)'"
+
+        case .error_displaying_offer_code_redemption_sheet_no_window_scene:
+            return "Could not display the Offer Code redemption sheet: could not " +
+            "determine the UIWindowScene to present the sheet over. Please try " +
+            "passing in the UIWindowScene that you'd like to present the sheet over."
+
+        case .error_displaying_offer_code_redemption_sheet_unavailable_in_app_extension:
+            return "Could not display the Offer Code redemption sheet: unavailable in iOS and Mac Catalyst" +
+            "App Extensions below iOS/Mac Catalyst 16."
+
+        case .not_displaying_offer_code_redemption_sheet_because_ios_app_on_macos:
+            return "Not displaying the Offer Code redemption sheet because this is an iOS app running on macOS," +
+            "and displaying an Offer Code redemption sheet as a 'Designed for iPad' app on macOS throws an error."
```

Comment:
I'd simplify a bit this message to remove the "Designed for iPad" part, since if the app only supports iPhone it would be "Designed for iPhone" (Xcode actually calls it this way).

Maybe:

```
Could not display the Offer Code redemption sheet: not supported in iOS apps running on macOS.
```

Reply:
Love it! Updated in 409dcac82be0af39eb817e2b861877d3be6cace6